{"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"identifier":{"url":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/errorData(_:)","interfaceLanguage":"swift"},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorData","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/AsyncMoya\/documentation\/AsyncMoya","doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError"]]},"variants":[{"paths":["\/documentation\/asyncmoya\/dataerror\/errordata(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"abstract":[{"type":"text","text":"서버로부터 받은 오류 메시지 문자열을 직접 저장할 때 사용합니다."}],"metadata":{"externalID":"s:9AsyncMoya9DataErrorO05errorC0yACSScACmF","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorData"},{"kind":"text","text":"("},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":")"}],"roleHeading":"Case","role":"symbol","modules":[{"name":"AsyncMoya"}],"symbolKind":"case","title":"DataError.errorData(_:)"},"kind":"symbol","references":{"doc://AsyncMoya/documentation/AsyncMoya":{"type":"topic","title":"AsyncMoya","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya","url":"\/documentation\/asyncmoya","abstract":[{"text":"Moya를 위한 현대적인 async\/await, Combine, RxSwift 확장 라이브러리","type":"text"}],"role":"collection","kind":"symbol"},"doc://AsyncMoya/documentation/AsyncMoya/DataError":{"navigatorTitle":[{"text":"DataError","kind":"identifier"}],"type":"topic","title":"DataError","kind":"symbol","url":"\/documentation\/asyncmoya\/dataerror","identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError","abstract":[{"type":"text","text":"네트워크 요청 및 응답 처리 중 발생할 수 있는 다양한 오류를 표현하는 열거형"}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"DataError","kind":"identifier"}]},"doc://AsyncMoya/documentation/AsyncMoya/DataError/errorData(_:)":{"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorData"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":")"}],"identifier":"doc:\/\/AsyncMoya\/documentation\/AsyncMoya\/DataError\/errorData(_:)","kind":"symbol","url":"\/documentation\/asyncmoya\/dataerror\/errordata(_:)","type":"topic","title":"DataError.errorData(_:)","abstract":[{"text":"서버로부터 받은 오류 메시지 문자열을 직접 저장할 때 사용합니다.","type":"text"}]}}}